CorsixTH/Lua/dialogs/resizables/file_browsers/load_game.lua

```diff
@@ -41,12 +41,16 @@ function UILoadGame:choiceMade(name)
   local status, err = pcall(app.load, app, name)
   if not status then
     err = _S.errors.load_prefix .. err
-    print(err)
-    app:loadMainMenu()
-    app.ui:addWindow(UIInformation(self.ui, {err}))
+    self:loadError(err)
```

Comment:
You moved from multiple uses of the updated `err` to one use of it in a function call. At that point a perhaps good alternative is not to assign to the variable, but instead move the entire expression into the call.